open-metadata-implementation/adapters/open-connectors/governance-action-connectors/pom.xml

```diff
@@ -50,10 +50,21 @@
             <artifactId>repository-services-apis</artifactId>
         </dependency>
 
-      <dependency>
+        <dependency>
            <groupId>commons-io</groupId>
            <artifactId>commons-io</artifactId>
-       </dependency>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <exclusions>
+                <exclusion>
```

Comment:
Out of interest, is this exclusion because you don't require the annotations, or did you hit a clash (the latter case has caused a latent issue in the past)?

Reply:
I copied it from another pom file - assumed you had added it - can take it out if not needed.